[metricbeat] Fix errors in process summary on latest Linux kernels

kesslerm opened the pull request:
On Linux 4.14 and 4.15, metricbeat logs large numbers of errors such as

```
2018/02/07 11:19:46.284926 process_summary.go:79: ERR Unknown state <73> for process with pid 4
```

All affected processes appear to be kernel threads.

`<73>` translates into `I`, which is a new status code introduced into the
`/proc/<pid>/stat` output in kernel commit
https://github.com/torvalds/linux/commit/06eb61844d841d0032a9950ce7f8e783ee49c0d0

For backwards compatibility, the old and new codes are now used to increment
the count of idle processes reported in the summary.

Fixes #6305.

ph commented:
@karmi This PR should pass the CLA check, @kesslerm is part of the elastic org?

ph commented:
@kesslerm Would you mind adding a changelog entry for metricbeat?

karmi commented:
@ph, there might be a problem with the [email address](https://github.com/elastic/beats/commit/03b83d78a03ad08c11ef00867ccc6e6c3966bd8e.patch) — @kesslerm, I think you've used a different email in the Git commit than is attached to your Github profile. If you add it there, it should work, that's the simplest way of fixing the check.

ph commented:
@karmi Ah! the email is not linked to his github! gotcha! thanks.

kesslerm commented:
@ph, @karmi, I've adjusted the commit email address and added a changelog entry before force-updating my branch. Please let me know if this is OK now.
